src/main/java/org/sonarlint/intellij/finding/LiveFinding.java

```diff
@@ -69,6 +76,17 @@ protected LiveFinding(Issue issue, PsiFile psiFile, @Nullable RangeMarker range,
     this.context = context;
     this.quickFixes = quickFixes;
     this.ruleDescriptionContextKey = issue.getRuleDescriptionContextKey().orElse(null);
+    this.cleanCodeAttribute = issue.getCleanCodeAttribute();
+
+    if(!issue.getImpacts().isEmpty()){
+      this.impacts = issue.getImpacts();
+    } else {
+      Map<SoftwareQuality, ImpactSeverity> defaultImpacts = new HashMap<>();
+      defaultImpacts.put(SoftwareQuality.MAINTAINABILITY, ImpactSeverity.HIGH);
+      defaultImpacts.put(SoftwareQuality.RELIABILITY, ImpactSeverity.MEDIUM);
+      defaultImpacts.put(SoftwareQuality.SECURITY, ImpactSeverity.LOW);
+      this.impacts = defaultImpacts;
```

Comment:
I think you can just do `this.impacts = Map.of(SoftwareQuality.MAINTAINABILITY, ImpactSeverity.HIGH, SoftwareQuality.RELIABILITY, ImpactSeverity.MEDIUM, SoftwareQuality.SECURITY, ImpactSeverity.LOW);`